mp/src/game/client/hl2/hud_credits.cpp

```diff
@@ -137,8 +120,50 @@ class CHudCredits : public CHudElement, public Panel
     float m_flCreditsPixelHeight;
 
     HFont m_hTFont;
+
+    int m_iMusicGUID;
+
+    ConVarRef sv_unlockedchapters;
 };
 
+using namespace vgui;
+
+static CHudCredits *g_pCreditsDialog = nullptr;
+
+//-----------------------------------------------------------------------------
+// Purpose: Constructor
+//-----------------------------------------------------------------------------
+CHudCredits::CHudCredits() : BaseClass(nullptr, "HudCredits"), sv_unlockedchapters("sv_unlockedchapters")
```

Comment:
Can rename this panel to remove the "HUD" references, as it is no longer a HUD element. I would suggest the name `CreditsDialog`.